src/videojs-http-streaming.js

```diff
@@ -335,6 +332,7 @@ class HlsHandler extends Component {
   setOptions_() {
     // defaults
     this.options_.withCredentials = this.options_.withCredentials || false;
+    this.options_.limitRenditionByPlayerDimensions = this.options_.limitRenditionByPlayerDimensions || true;
```

Comment:
Wont this always be true?

Reply:
![https://media3.giphy.com/media/14aUO0Mf7dWDXW/giphy.gif](https://media3.giphy.com/media/14aUO0Mf7dWDXW/giphy.gif)